File: JavaWeb2-api/src/main/java/com/javaweb/web/eo/role/RoleListRequest.java
package com.javaweb.web.eo.role;

import java.io.Serializable;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class RoleListRequest implements Serializable {
	
	private static final long serialVersionUID = -250839408653592074L;

	private String roleName;//角色名称
	
	private String createStartDate;//角色创建的开始日期
	
	private String createEndDate;//角色创建的结束日期
	
	private Long currentPage = 1L;//默认当前第1页
	
	private Long pageSize = 10L;//默认每页显示10条
	
}

File: JavaWeb2-angular/src/app/model/role/RoleListRequest.ts
import {CommonConstant} from '../../constant/CommonConstant';

export class RoleListRequest {

  roleName:string = CommonConstant.EMPTY;//角色名

  createStartDate:any = null;//角色创建的开始日期

  createEndDate:any = null;//角色创建的结束日期

  currentPage = 1;//当前页数

  pageSize = 10;//每页显示多少条

}
